.github/workflows/ci.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       uses: actions/cache@v2
       with:
         path: dependencies/vtk_install
-        key: vtk-${{matrix.vtk_version}}-${{matrix.os}}
+        key: vtk-build-${{matrix.vtk_version}}-${{matrix.os}}
```

Comment:
shouldn't you restore the key name?

Reply:
If I change it back, the old cache will be used. Not a big deal imo as long as the release as all the correct needed flags.